### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -113,6 +109,7 @@ const staticify = (root, options) => {
     };
 
     const serve = req => {
+        // eslint-disable-next-line node/no-deprecated-api
         const filePath = stripVersion(url.parse(req.url).pathname);
```
Comment (XhmikosR) [inline index.js]: I couldn't make this work with relative URLs so for now I just disabled the error. We should definitely revisit, or if anyone has a working solution let me know.

Comment (rakeshpai) [reply index.js]: I think the `url.parse(req.url).pathname` needs to change to `(new URL(req.url)).pathname`. Worth trying it to be sure.

Comment (XhmikosR) [reply index.js]: That returns `undefined`, I already tried it.

Comment (XhmikosR) [reply index.js]: We need to pass the base option for this to work...

**Comment by rakeshpai** [comment]
I can merge, but coveralls is complaining (a whooping -0.3%!). Shall I just merge?

**Comment by XhmikosR** [comment]
Yeah, it's OK in this case. I will make an issue about `URL`.

**Pull request merged**